src/lib/components/ui/input/input.svelte

```diff
@@ -50,9 +50,11 @@
 </script>
 
 <div class="root {layout} {dimension} {className}">
-	<label class="label" for={labelFor}>
-		{label}
-	</label>
+	{#if label !== ''}
+		<label class="label" for={labelFor}>
+			{label}
+		</label>
+	{/if}
```

Comment:
I think `label` by default is `undefined` not an empty string `''`. Maybe this should be just:
```suggestion
	{#if label}
		<label class="label" for={labelFor}>
			{label}
		</label>
	{/if}
```